src/query.c

```diff
@@ -216,6 +216,7 @@ simplet_status_t simplet_query_process(simplet_query_t *query,
   if (!(olayer = OGR_DS_ExecuteSQL(source, query->ogrsql, NULL, NULL))) {
     int err = CPLGetLastErrorNo();
     if (!err) {
+      // FIXME: This should be a problem, but it is ignored right now.
```

Comment:
I've created https://github.com/propublica/simple-tiles/issues/21 in response to this.